### DISCUSSION TITLE
Only show items for some statuses

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-checklists#462] [Feature Requests]
Let's add a new column: "Statuses". This is related to this thread:
https://github.com/publishpress/PublishPress-Checklists/issues/108

In this column, users can choose which statuses the checklist should show for.

- All statuses should be loaded by default
- Users can remove statuses. If the remove a status, the checklist item will not show for them

![Screen Shot 2021-04-23 at 12 29 45 PM](https://user-images.githubusercontent.com/3868028/115901825-b8749b80-a42f-11eb-820e-9098c5950548.png)

**Comment by stevejburge** [comment]
This will be a Pro feature

**Comment by andergmartins** [comment]
Hi @julianactrl, thanks for the contribution. Please, don't send changes to the development. I have moved your commits to a new branch: `feature-270-status-filter-move-to-pro`.

I was testing and it works well, except for the fact it doesn't update after saving to another status. It only works after page refreshing. Here is a test case:

1. Open the checklists and add "Pending" to the "Number of characters in the title" (which should be Recommended or Required).
2. Save changes
3. Create a new post
4. Verify that the item "Number of characters in the title" is not on the checklist. Good.
5. Add a title and comment to the post. Save as a Draft.
6. Go to the Post tab in the editor, and check the box: "Pending review"
7. Save as Pending Review
8. Verify that the item "Number of characters in the title" is not in the list. Here is the problem.
9. Refresh the page. Verify it will display correctly.

Please, could you fix this? Let us know if you have any questions.

Thanks

**Comment by julianactrl** [comment]
Hi @andergmartins , thanks for your review.

I will fix it and will let you know.

Thanks again.

**Comment by stevejburge** [comment]
Solved with 2.19
